SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +128,68 @@ etc...
 
     > These lines ensure that the SpatialOS Unreal GDK can override Unreal's network replication.
     >
```

Comment:
I can't comment on it as it's not an edited part of the page but the line:
"These lines ensure that the SpatialOS Unreal GDK can override Unreal's network replication."
-> 
"These lines ensure that the GDK can override Unreal's network replication."